src/indieweb_utils/replies/context.py

```diff
@@ -7,320 +9,302 @@
 
-            if h_card["properties"].get("name"):
-                author_name = h_card["properties"]["name"][0]
-            else:
-                author_name = None
-
-            if h_card["properties"].get("photo"):
-                author_image = h_card["properties"]["photo"][0]
-                if author_image.startswith("//"):
-                    author_image = "https:" + author_image
-                elif author_image.startswith("/"):
-                    author_image = url.split("/")[0] + "//" + domain + author_image
-                elif author_image.startswith("http://") or author_image.startswith("https://"):
-                    author_image = author_image
+                if author["items"][0]["properties"].get("name"):
+                    author_name = h_entry["properties"]["author"][0]["properties"]["name"][0]
                 else:
-                    author_image = "https://" + domain + "/" + author_image
-            else:
-                author_image = None
+                    author_name = ""
 
-            if h_card["properties"].get("note"):
-                post_body = h_card["properties"]["note"][0]
-            else:
-                post_body = None
+                if author["items"]["properties"].get("photo"):
+                    author_image = h_entry["properties"]["author"][0]["properties"]["photo"][0]
+                else:
+                    author_image = ""
 
-            h_entry = {
-                "type": "entry",
-                "author": {"type": "card", "url": url, "name": author_name, "photo": author_image},
-                "url": url,
-                "content": {"html": post_body},
-            }
+        if author_url is not None and author_url.startswith("/"):
+            author_url = parsed_url.scheme + "://" + domain + author_url
 
-            return True, h_entry, site_supports_webmention
+        if author_image is not None and author_image.startswith("/"):
+            author_image = parsed_url.scheme + "://" + domain + author_image
 
-        h_entry = {}
+    if h_entry["properties"].get("content") and h_entry["properties"].get("content")[0].get("html"):
+        post_body = h_entry["properties"]["content"][0]["html"]
+        soup = BeautifulSoup(post_body, "html.parser")
+        post_body = soup.text
 
-        if parsed_url.netloc == "twitter.com" and twitter_bearer_token is not None:
-            site_supports_webmention = False
-            tweet_uid = url.strip("/").split("/")[-1]
+        favicon = soup.find("link", rel="icon")
 
-            headers = {"Authorization": f"Bearer {twitter_bearer_token}"}
+        if favicon and not author_image:
+            photo_url = favicon["href"]
+            if not photo_url.startswith("https://") or not photo_url.startswith("http://"):
+                author_image = "https://" + domain + photo_url
+        else:
+            author_image = ""
+
+        post_body = " ".join(post_body.split(" ")[:75]) + " ..."
```

Comment:
75 should probably be passed in as a kwarg to this function so it can be overridden, but it's ok as is for now.